src/main/java/com/atlassian/shele/shelePlugin/ao/ProjectEntityService.java

```diff
@@ -22,15 +22,17 @@ public ProjectEntityService(@ComponentImport ActiveObjects activeObjects, Projec
 
     public List<ProjectDTO> saveEntity(ProjectDTO dto) {
         List<ProjectEntity> projectEntityList = new ArrayList<>();
-        List<String> list = dto.getProject();
-        List<Long> longs = dto.getEventTypeIds();
-        for (String s : list) {
+        List<String> stringList = dto.getProject();
+        List<Long> longList = dto.getEventTypeIds();
+        String convertString = String.valueOf(longList);
+        stringList.forEach(s -> {
```

Comment:
it is better to rewrite it using map function like:
```
        List<ProjectEntity> projectEntityList = stringList.stream().map(s -> {
            ProjectEntity entity = activeObjects.create(ProjectEntity.class);
            entity.setProject(s);
            entity.setEventTypeId(convertString);
            entity.save();
            return entity;
        }).collect(toList());
```

also, lambda expression could be moved to a separate function and the expression will be easier to read:
```
        List<ProjectEntity> projectEntityList = stringList.stream()
                .map(key -> createProjectEntity(eventTypeIds, key))
                .collect(toList());
```